### PR TITLE
Audit logging

### PR DESCRIPTION
**Pull request by sunithasomanna**
Added a new section "Audit logging" in the API readme.

### CODE REVIEW

**Review by shivacharanms — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -11028,3 +11029,17 @@ curl -i -X PATCH \
 
 
 
+# Audit Logging
+
+Audit logs has information on each API and is stored in the API.log file.  Each log consists of a code number followed by the date and time of the log, hostname from which the APIs were sent, user account and role details, API request method and resource, response body, response code and the message.
```
Comment (amar-shalgar) [inline docs/README.md]: API.log should be api.log under odimra logs

File: docs/README.md [hunk 2]
```diff
@@ -11028,3 +11029,17 @@ curl -i -X PATCH \
...
+
+**Samples**
+
+- <110> 2009-11-10T23:00:00Z 10.24.1.237 [account@1 user="admin" roleID="Administrator"][request@1 method="GET" resource="/redfish/v1/Systems" requestBody=""][response@1 responseCode=200] Operation Successful
+
+- <110> 2022-01-17T13:57:48Z 10.24.1.237 [account@1 user="admin" roleID="Administrator"][request@1 method="POST" resource="/redfish/v1/AggregationService/AggregationSources" requestBody="{"HostName":"10.24.0.5","Links":{"ConnectionMethod":{"@odata.id":"/redfish/v1/AggregationService/ConnectionMethods/337ea3cb-3acc-49e2-b33f-3f5ce2a5ada4"}},"Password":"null","UserName":"admin"}"][response@1 responseCode=202] Operation successful
+
+- <107> 2009-11-10T23:00:00Z 10.24.1.237 [account@1 user="admin" roleID="Administrator"][request@1 method="GET" resource="/redfish/v1/Systems" requestBody=""][response@1 responseCode=404] Operation failed
+
+  <blockquote> Note: <110> is the audit information code and <107> is the audit error code. </blockquote>
```
Comment (amar-shalgar) [inline docs/README.md]: <110> and <107> are the Priority value 110 indicates audit info log and 107 indicates audit error log

**Review by amar-shalgar — CHANGES_REQUESTED** [review]
Please address the changes

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
Do not merge into dev branch till we get approval for this.

File: docs/README.md (hunk 2 above)
Comment (sunithasomanna) [reply docs/README.md]: done

File: docs/README.md (hunk 1 above)
Comment (sunithasomanna) [reply docs/README.md]: done

File: docs/README.md [hunk 3]
```diff
@@ -11028,3 +11029,17 @@ curl -i -X PATCH \
 
 
 
+# Audit Logging
+
+Audit logs has information on each API and is stored in the api.log file in odimra logs.  Each log consists of a priority value followed by the date and time of the log, hostname from which the APIs were sent, user account and role details, API request method and resource, response body, response code and the message.
+
+**Samples**
+
+- <110> 2009-11-10T23:00:00Z 10.24.1.237 [account@1 user="admin" roleID="Administrator"][request@1 method="GET" resource="/redfish/v1/Systems" requestBody=""][response@1 responseCode=200] Operation Successful
```
Comment (shivacharanms) [inline docs/README.md:11038]: Change the IPs everywhere to opensource IPs, eg: 192.168.100.101

Comment (shivacharanms) [reply docs/README.md:11038]: Take this up as a separate change in a new PR with IP changes across all references

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**